manifest.json

```diff
@@ -1,11 +1,19 @@
 {
     "manifest_version": 2,
-
     "name": "iOS Status Bar Overlay",
     "description": "This extension adds an iOS status bar overlay on the current page.",
     "version": "1.0",
     "content_scripts": [{
-        "matches": ["http://localhost:*/*"],
+        "matches": ["http://localhost:*/*", "http://local.*.*:*/*"],
```

Comment:
If I delete the second entry then it works. If we have a button do we even need this though?